cmd/plugin-backend.go

```diff
@@ -18,6 +18,7 @@ var (
 	staticPathArg   = flag.String("static-path", "", "static files path to serve frontend (default: './web/dist')")
 	configPathArg   = flag.String("config-path", "", "config files path (default: './config')")
 	pluginConfigArg = flag.String("plugin-config-path", "", "plugin yaml configuration")
+	logVerbosityArg = flag.Int("log-verbosity", 0, "verbosity of logs (0 for low, 1 for high)")
```

Comment:
it might be better to use a standard `log.level` flag with string values. These values can be set up directly in logrus, using something like:
```
level, err := logrus.ParseLevel(logLevel)
	if err != nil {
		logrus.WithError(err).Fatal("unable to set the log level")
	}
	logrus.SetLevel(level)
	```

Reply:
I swapped to using the log-level strings like you had suggested. Since the request level logging is being done through a separate package, just changing the logrus level didn't prevent it, so that part of the code has been kept the same. I changed it instead to look for the "trace" level since that seemed the most apt.